docs/reference/cli/resource/list.md

```diff
@@ -47,9 +47,9 @@ type                       version tags                        description
 ----                       ------- ----                        -----------
 Test/TestGroup             0.1.0
 Microsoft/OSInfo           0.1.0   {os, linux, windows, macos} Returns information about the operating system.
-Microsoft.Windows/Registry 0.1.0   {Windows, NT}               Registry configuration provider for the Windows Registry
+Microsoft.Windows/Registry 0.1.0   {Windows, NT}               Registry configuration adapter for the Windows Registry
```

Comment:
This should match the description string for the registry resource, which I recommended changing to:

```text
Manage Windows Registry keys and values
```